package.json

```diff
@@ -31,6 +31,7 @@
     "onchange": "^3.2.1",
     "postcss-cli": "^4.1.0",
     "stylelint": "^8.0.0",
+    "stylelint-config-nebula": "^0.2.0",
```

Comment:
You will want to remove the following from your package.json as they are now dependencies within `styelint-config-nebula`

```
"stylelint-config-sass-guidelines": "^2.2.0",
"stylelint-scss": "^1.5.1",
"stylelint-selector-bem-pattern": "^1.1.0"
```